### ISSUE TITLE
Alternatives for asyncio.create_subprocess_exec method for nuitka.

### ISSUE THREAD

**Issue by thesct22** [Nuitka/Nuitka#1952]
## Nuitka Version

```
1.3rc9
Commercial: None
Python: 3.8.8 (tags/v3.8.8:024d805, Feb 19 2021, 13:18:16) [MSC v.1928 64 bit (AMD64)]
Flavor: CPython Official
Executable: C:\Soultion-Applications\d-test\.venv\Scripts\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
Version C compiler: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30037\bin\HostX64\x64\cl.exe (cl 14.2).
```

## Packages

```
Nuitka==1.3rc9
orderedset==2.0.3
```

## Nutika command used

```
python -m nuitka --standalone main.py
```

## main.py

```python
import asyncio
import sys
from pathlib import Path


async def main():
    this_source_file = Path(__file__)
    main_dir = this_source_file.parent
    runner_main_file = main_dir / "subproc.py"
    if not runner_main_file.is_file():
        raise RuntimeError(
            "Unable to find the file./n"
            f"Expecting to find '{runner_main_file}'."
        )
    args = [str(runner_main_file)]

    process = await asyncio.create_subprocess_exec(
        sys.executable,
        *args,
        stdout=asyncio.subprocess.PIPE,
        stderr=asyncio.subprocess.PIPE,
    )

    stdout, stderr = await process.communicate()

    print(f'[{process.returncode}]')
    if stdout:
        print(f'[stdout] {stdout.decode()}')
    if stderr:
        print(f'[stderr] {stderr.decode()}')

asyncio.run(main())
```

## subproc.py

```python
import time

time.sleep(3)
print('Hello from subproc.py after 3 seconds')
```

## Issue
* Is there a way to run whatever is inside subproc.py without having to create a subproc.exe and then calling it? subproc.py has circular dependency on some methods inside modules which calls main.py. So this option cannot be used.
* subproc.py should also be obfuscated. So cannot have subproc.py in the dist.

**Comment by kayhayen** [comment]
You will have to wait for the so called `multidist` feature, which is on the roadmap. It will allow combining arbitrarily many programs into a single binary.

**Comment by kayhayen** [comment]
Multidist has been implemented and release with I think 1.4 already. Give it a shot.
